lib/Hook/hook.cpp

```diff
@@ -36,10 +36,21 @@ void hook::calculateAndDrawHook(int xOrigin, int yOrigin, float angle, Item item
                     int yCircle = yStopMoment - (int)(i2 * sin(angle));
 
                     c.drawItemWhenGrabbed(xCircle, yCircle, item.size, item.type);
-                    _delay_ms(50);
+
+                    if(item.size < 10) {
+                        _delay_ms(3);
+                    } else if (item.size < 20) {
+                        _delay_ms(10);
+                    } else if (item.size < 30) {
+                        _delay_ms(20);
+                    } else if (item.size < 40) {
+                        _delay_ms(40);
+                    } else {
+                        _delay_ms(80);
+                    }
+
```

Comment:
make this a function like item.size/3. so you have a solution that is more gradual and works for all sizes.

Reply:
hou je bek

Reply:
:(